Tests/BugsnagSwiftConfigurationTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BugsnagSwiftConfigurationTests.swift
+//  Tests
+//
+//  Created by Robin Macharg on 22/01/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import XCTest
+
+class BugsnagSwiftConfigurationTests: XCTestCase {
+
+    /**
+     * Objective C trailing-NSError* initializers are translated into throwing
+     * Swift methods, allowing us to fail gracefully (at the expense of a more-explicit
+     * (read: longer) ObjC invocation).
+     */
+    func testDesignatedInitializerHasCorrectNS_SWIFT_NAME() {
+        
+        do {
+            let _ = try BugsnagConfiguration(DUMMY_APIKEY_16CHAR)
```

Comment:
If this line "succeeds" then the test still passes though the behavior is broken. An `XCTFail()` assertion after this line would catch that.